src/vcBPA.cpp

```diff
@@ -923,87 +920,94 @@ vdkError vcBPA_ConvertReadPoints(vdkConvertCustomItem *pConvertInput, vdkPointBu
   // Reset point count to avoid infinite loop
   pBuffer->pointCount = 0;
 
-  vcBPAConvertItem *pData = (vcBPAConvertItem *)pConvertInput->pData;
-  bool getNextGrid = true;
+  while (pBuffer->pointCount == 0)
```

Comment:
What bug does this fix?

Reply:
with the exclusion of points outside the AABB of a grid it's possible that grids are present with 0 points, this prevents early termination of the conversion by proceeding to subsequent grids